tools/monitoring/environments/default/main.jsonnet

```diff
@@ -1 +1,2 @@
-import 'monitoring.libsonnet'
```

Comment:
This files is wiped on every deploy you should add to `monitoring.libsonnet` instead